ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-tech-support.j2

```diff
@@ -0,0 +1,40 @@
+{# Management Tech-Support #}
+{% if management_tech_support is arista.avd.defined %}
+
+## Management Tech-Support
+
```

Comment:
Please remove this blank line and add it above the next heading (line7). We add blank lines before heading and this makes sure the blank lines are added only if a new heading will be added. Please do the same for all headings in the template.